lib/Serialization/ModuleFormat.h

```diff
@@ -1703,6 +1703,12 @@ namespace decls_block {
     BCVBR<5>  // index
   >;
 
+  using XRefClangTemplateInstantiationLayout = BCRecordLayout<
```

Comment:
Do we need to bump `SWIFTMODULE_VERSION_MINOR`?

Reply:
Done